app/dashboard/src/components/dashboard/AssetRow.tsx

```diff
@@ -250,25 +253,25 @@ export const AssetRow = React.memo(function AssetRow(props: AssetRowProps) {
     [doDeleteRaw, item.item],
   )
 
-  const doTriggerDescriptionEdit = React.useCallback(() => {
+  const doTriggerDescriptionEdit = useEventCallback(() => {
     setModal(
       <EditAssetDescriptionModal
         doChangeDescription={async (description) => {
           if (description !== asset.description) {
             setAsset(object.merger({ description }))
 
-            await backend
-              .updateAsset(item.item.id, { parentDirectoryId: null, description }, item.item.title)
-              .catch((error) => {
-                setAsset(object.merger({ description: asset.description }))
-                throw error
-              })
+            // eslint-disable-next-line no-restricted-syntax
```

Comment:
what is the restricted syntax being used here?

Reply:
Early returns. Maybe we should allow them?

Reply:
ah. possibly, yeah